airbyte-integrations/bases/base-normalization/oracle.Dockerfile

```diff
@@ -43,6 +43,7 @@ RUN pip install dbt-oracle==0.4.3
 
 WORKDIR /airbyte/normalization_code/dbt-template/
 # Download external dbt dependencies
```

Comment:
Watching this PR just to see when I can merge (dont take this as pressure)

But I wanted to chime in here to say it would be great to leave a link to the dbt github issue as to why we did this.

Thanks for working hard to unblock ci btw 🙏

Reply:
Added a link to the DBT issue. Good call.